builder/vmware/iso/step_create_vmx.go

```diff
@@ -110,14 +367,175 @@ func (s *stepCreateVMX) Run(state multistep.StateBag) multistep.StepAction {
 		}
 	}
 
-	ctx.Data = &vmxTemplateData{
+	templateData := vmxTemplateData{
 		Name:     config.VMName,
 		GuestOS:  config.GuestOSType,
 		DiskName: config.DiskName,
 		Version:  config.Version,
 		ISOPath:  isoPath,
+
+		SCSI_Present:         "FALSE",
+		SCSI_diskAdapterType: "lsilogic",
+		SATA_Present:         "FALSE",
+		NVME_Present:         "FALSE",
+
+		DiskType:              "scsi",
+		CDROMType:             "ide",
```

Comment:
Also here - I think it makes more sense to have the CDROMType mirror the disk type (in this case, SCSI) where possible right? So SCSI disk = SCSI CDROM, SATA disk = SATA CDROM, etc.

Reply:
I'll try messing with it maybe next weekish when I have more time. I'm currently focused on another one of my PRs being merged upstream and this Spectre vulnerability hit the public which I'm being asked to weaponize and so that's occupying most of my time this weekend.

Reply:
Ok. So I exposed the cdrom adapter type to the user so that they can choose which adapter type they want as `cdrom_adapter_type` like you specified your prior comment.